accelerators/stratus_hls/sort_stratus/hw/hls/project.tcl

```diff
@@ -1,5 +1,3 @@
-#  Copyright (c) 2011-2023 Columbia University, System Level Design Group
-#  SPDX-License-Identifier: Apache-2.0
 
```

Comment:
can you add back the copyright?

Reply:
I haven't removed it. I will add them back anyways. There was a command to do/update them automatically, right!?

Reply:
KL has a script that does all the files, but I think since you have just 3, you can do it manually